common/registrar/registrar.go

```diff
@@ -55,10 +55,10 @@ So the caller needs to make sure the relevant environment initialised the desire
 contracts
```

Comment:
Now the addresses of the 3 contracts are hard coded and the contracts are deployed these docs can be updated?